### PR TITLE
Fix letter-frequency.md: remove wrong spaces and make quotes consistent

### PR DESCRIPTION
**Pull request by melusc**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

In the tests the spaces aren't there but they were in the test description.

Remove spaces that are not meant to be there (L37, L134, L156)
Make quotes consistent across the file (L61)

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/letter-frequency.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ assert.deepEqual(letterFrequency('Not all that Mrs. Bennet, however'), [
 ]);
 ```
 
-`letterFrequency("daughters, could ask on the ")` should return `[[' ',5],[',',1],['a',2],['c',1],['d',2],['e',2],['g',1],['h',2],['k',1],['l',1],['n',1],['o',2],['r',1],['s',2],['t',2],['u',2]]`.
+`letterFrequency("daughters, could ask on the ")` should return `[[" ",5],[",",1],["a",2],["c",1],["d",2],["e",2],["g",1],["h",2],["k",1],["l",1],["n",1],["o",2],["r",1],["s",2],["t",2],["u",2]]`.
```
Comment (moT01) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/letter-frequency.md:61]:
```suggestion
`letterFrequency("daughters, could ask on the ")` should return `[[" ", 5],[",", 1],["a", 2],["c", 1],["d", 2],["e", 2],["g", 1],["h", 2],["k", 1],["l", 1],["n", 1],["o", 2],["r", 1],["s", 2],["t", 2],["u", 2]]`.
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Since this is a formatting PR, I noticed one that could use some spacing to align with the rest.

**Review by moT01 — APPROVED** [review]
LGTM @melusc 🎉

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Comment by nhcarrigan** [comment]
Thank you for your continued contribution to our project. We have reviewed your pull request and are happy to accept these changes.

Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.

**Pull request merged**